slicerio/server.py

```diff
@@ -173,33 +205,35 @@ def file_load(file_path, file_type=None, properties=None, auto_start=True, timeo
     if file_type is None:
         file_type = "VolumeFile"
     p = urllib.parse.urlparse(file_path)
-    if p.scheme == 'slicer':
+    if p.scheme == "slicer":
         # Slicer URL - use it as is. For example:
         # slicer://viewer/?studyUID=1.2.826.0.1.3680043.8.498.77209180964150541470378654317482622226&dicomweb_endpoint=http%3A%2F%2F130.15.7.119:2016%2Fdicom&bulk_retrieve=0
-        url_encoded_path = urllib.request.quote(file_path, safe='')
+        url_encoded_path = urllib.request.quote(file_path, safe="")
         api_url = f"http://127.0.0.1:{SERVER_PORT}/slicer/open?url={url_encoded_path}"
     else:
         # Local file path or remote download path
-        path_type = 'url' if p.scheme in ['http', 'https'] else 'localfile'
-        url_encoded_path = urllib.request.quote(file_path, safe='')
+        path_type = "url" if p.scheme in ["http", "https"] else "localfile"
+        url_encoded_path = urllib.request.quote(file_path, safe="")
         api_url = f"http://127.0.0.1:{SERVER_PORT}/slicer/mrml?{path_type}={url_encoded_path}&filetype={file_type}"
         if properties:
             for key in properties:
-                url_encoded_key = urllib.request.quote(key.encode(), safe='')
-                url_encoded_value = urllib.request.quote(str(properties[key]).encode(), safe='')
+                url_encoded_key = urllib.request.quote(key.encode(), safe="")
+                url_encoded_value = urllib.request.quote(
+                    str(properties[key]).encode(), safe=""
+                )
                 api_url += f"&{url_encoded_key}={url_encoded_value}"
 
     retry_after_starting_server = True
     try:
         response = requests.post(api_url)
         retry_after_starting_server = False
-    except requests.exceptions.ConnectionError as e:
+    except requests.exceptions.ConnectionError:
         if not auto_start:
             raise
 
     if retry_after_starting_server:
         # Try again, with starting a server first
-        server_process = start_server(slicer_executable)
+        server_process = start_server(slicer_executable)  # noqa: F841
```

Comment:
Why the noqa?

Reply:
`server_process` is not used

Reply:
much less noise without the formatting ...

Reply:
Ok, you can then it is better to delete `server_process = `